webui/react/src/services/decoder.ts

```diff
@@ -719,6 +719,14 @@ export const decodeTrialWorkloads = (
   };
 };
 
+export const decodeWorkspaceNamespaceBindings = (
```

Comment:
will this function use multiple places? if not, i think no need to defined func

Reply:
removed